go/framework/bertybridge/bridge_protocol.go

```diff
@@ -201,7 +202,7 @@ func newProtocolBridge(ctx context.Context, logger *zap.Logger, config *Messenge
 				SwarmAddrs:        defaultSwarmAddrs,
 				APIAddrs:          defaultAPIAddrs,
 				APIConfig:         APIConfig,
-				ExtraLibp2pOption: libp2p.ChainOptions(libp2p.Transport(mc.ProximityTransportConstructor(ctx, logger))),
```

Comment:
blocking until we merge #2499

Reply:
:'(

Reply:
feel free to rebase #2499 against your PR right now, to make things faster to merge later :)